src/test/java/org/jenkinsci/plugins/gitclient/WorkspaceWithRepo.java

```diff
@@ -72,7 +73,7 @@ private void renameAndDeleteDir(Path srcDir, String destDirName) {
      * Returns path to the local mirror directory.
      *
      * @return path to the local mirrror directory
-     * @throws IOException on I/O error
+     * @throws IOException          on I/O error
```

Comment:
Please don't make whiitespace changes to existing files.  It creates unnecessary barriers to other pull requests.
```suggestion
     * @throws IOException on I/O error
```